_saas-integrations/pinterest-ads/v1/pinterest-ads-v1.md

```diff
@@ -0,0 +1,150 @@
+---
+# -------------------------- #
+#     USING THIS TEMPLATE    #
+# -------------------------- #
+
+## NEED HELP USING THIS TEMPLATE? SEE:
+## https://docs-about-stitch-docs.netlify.com/reference/integration-templates/saas/
+## FOR INSTRUCTIONS & REFERENCE INFO
+
+
+# -------------------------- #
+#      Page & Formatting     #
+# -------------------------- #
+
+title: Pinterest Ads (vVERSION)
+permalink: /integrations/saas/pinterest-ads ## Add if there are multiple versions: /vVERSION
+keywords: pinterest-ads, integration, schema, etl pinterest-ads, pinterest-ads etl, pinterest-ads schema
+layout: singer
+# input: false
+
+key: "pinterest-ads-setup"
+
+
+# -------------------------- #
+#         Tap Details        #
+# -------------------------- #
+
+name: "pinterest-ads"
+display_name: "Pinterest Ads"
+
+singer: true
+status-url: ""
+
+tap-name: "Pinterest Ads" ## Ex: Intercom, not intercom
+repo-url: https://github.com/singer-io/tap-pinterest-ads
```

Comment:
We may need to comment this out, since the repo may not be public when the integration is released.